### ISSUE TITLE
Android app crashes after initializing TessBaseAPI

### ISSUE THREAD

**Issue by nicemanis** [bytedeco/javacv#1151]
The app crashes when `new TessBaseAPI()` is called.

I'm using the following version:
`implementation group: 'org.bytedeco.javacpp-presets', name: 'tesseract', version: '4.0.0-1.4.4'`

Here is the error log:
```
2019-03-06 16:36:45.090 19171-19171/caiaii.fingerreader E/AndroidRuntime: FATAL EXCEPTION: main
    Process: caiaii.fingerreader, PID: 19171
    java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/base.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_dependencies_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_resources_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_0_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_1_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_2_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_3_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_4_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_5_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_6_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_7_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_8_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/lib/arm, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/base.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_dependencies_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_resources_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_0_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_1_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_2_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_3_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_4_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_5_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_6_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_7_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_8_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_9_apk.apk!/lib/armeabi-v7a, /system/lib]]] couldn't find "libjnilept.so"
        at java.lang.Runtime.loadLibrary0(Runtime.java:1012)
        at java.lang.System.loadLibrary(System.java:1669)
        at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:1302)
        at org.bytedeco.javacpp.Loader.load(Loader.java:1043)
        at org.bytedeco.javacpp.Loader.load(Loader.java:935)
        at org.bytedeco.javacpp.lept.<clinit>(lept.java:10)
        at java.lang.Class.classForName(Native Method)
        at java.lang.Class.forName(Class.java:453)
        at org.bytedeco.javacpp.Loader.load(Loader.java:994)
        at org.bytedeco.javacpp.Loader.load(Loader.java:935)
        at org.bytedeco.javacpp.tesseract$TessBaseAPI.<clinit>(tesseract.java:3682)
        at caiaii.fingerreader.ImageAnalyzer.<init>(ImageAnalyzer.java:25)
        at caiaii.fingerreader.ImageAnalyzer.getInstance(ImageAnalyzer.java:45)
        at caiaii.fingerreader.MainActivity.onPermissionsGranted(MainActivity.java:91)
        at caiaii.fingerreader.MainActivity.checkPermissions(MainActivity.java:73)
        at caiaii.fingerreader.MainActivity.onCreate(MainActivity.java:34)
        at android.app.Activity.performCreate(Activity.java:7149)
        at android.app.Activity.performCreate(Activity.java:7140)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1288)
2019-03-06 16:36:45.091 19171-19171/caiaii.fingerreader E/AndroidRuntime:     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3017)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3172)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1906)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6863)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/base.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_dependencies_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_resources_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_0_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_1_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_2_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_3_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_4_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_5_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_6_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_7_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_8_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/lib/arm, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/base.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_dependencies_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_resources_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_0_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_1_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_2_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_3_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_4_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_5_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_6_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_7_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_8_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_9_apk.apk!/lib/armeabi-v7a, /system/lib]]] couldn't find "liblept.so"
        at java.lang.Runtime.loadLibrary0(Runtime.java:1012)
        at java.lang.System.loadLibrary(System.java:1669)
        at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:1302)
        at org.bytedeco.javacpp.Loader.load(Loader.java:1021)
        	... 27 more
2019-03-06 16:36:45.092 19171-19171/caiaii.fingerreader E/AndroidRuntime: FATAL EXCEPTION: main
    Process: caiaii.fingerreader, PID: 19171
    java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/base.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_dependencies_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_resources_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_0_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_1_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_2_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_3_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_4_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_5_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_6_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_7_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_8_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/lib/arm, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/base.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_dependencies_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_resources_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_0_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_1_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_2_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_3_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_4_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_5_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_6_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_7_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_8_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_9_apk.apk!/lib/armeabi-v7a, /system/lib]]] couldn't find "libjnilept.so"
        at java.lang.Runtime.loadLibrary0(Runtime.java:1012)
        at java.lang.System.loadLibrary(System.java:1669)
        at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:1302)
        at org.bytedeco.javacpp.Loader.load(Loader.java:1043)
        at org.bytedeco.javacpp.Loader.load(Loader.java:935)
        at org.bytedeco.javacpp.lept.<clinit>(lept.java:10)
        at java.lang.Class.classForName(Native Method)
        at java.lang.Class.forName(Class.java:453)
        at org.bytedeco.javacpp.Loader.load(Loader.java:994)
        at org.bytedeco.javacpp.Loader.load(Loader.java:935)
        at org.bytedeco.javacpp.tesseract$TessBaseAPI.<clinit>(tesseract.java:3682)
        at caiaii.fingerreader.ImageAnalyzer.<init>(ImageAnalyzer.java:25)
        at caiaii.fingerreader.ImageAnalyzer.getInstance(ImageAnalyzer.java:45)
        at caiaii.fingerreader.MainActivity.onPermissionsGranted(MainActivity.java:91)
        at caiaii.fingerreader.MainActivity.checkPermissions(MainActivity.java:73)
        at caiaii.fingerreader.MainActivity.onCreate(MainActivity.java:34)
        at android.app.Activity.performCreate(Activity.java:7149)
        at android.app.Activity.performCreate(Activity.java:7140)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1288)
2019-03-06 16:36:45.092 19171-19171/caiaii.fingerreader E/AndroidRuntime:     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3017)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3172)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1906)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6863)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/base.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_dependencies_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_resources_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_0_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_1_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_2_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_3_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_4_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_5_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_6_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_7_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_8_apk.apk", zip file "/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/lib/arm, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/base.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_dependencies_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_resources_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_0_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_1_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_2_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_3_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_4_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_5_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_6_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_7_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_8_apk.apk!/lib/armeabi-v7a, /data/app/caiaii.fingerreader-OWvUpNyjcTQAfvI9kN6EzA==/split_lib_slice_9_apk.apk!/lib/armeabi-v7a, /system/lib]]] couldn't find "liblept.so"
        at java.lang.Runtime.loadLibrary0(Runtime.java:1012)
        at java.lang.System.loadLibrary(System.java:1669)
        at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:1302)
        at org.bytedeco.javacpp.Loader.load(Loader.java:1021)
        	... 27 more
```

**Comment by saudet** [comment]
Try to use "tesseract-platform" instead of just "tesseract".

**Comment by nicemanis** [comment]
If I use tesseract-platform I get the following error:
`More than one file was found with OS independent path 'org/bytedeco/javacpp/linux-x86_64/libgomp.so.1'`

**Comment by saudet** [comment]
We can exclude those files, they're not for Android.

**Comment by saudet** [comment]
There are some sample projects here if you need those:
https://github.com/bytedeco/sample-projects/

**Comment by nicemanis** [comment]
Managed to fix the `More than one file was found with OS independent path` problem with the fix from #353 
```
android {
    packagingOptions {
        pickFirst 'org/bytedeco/javacpp/linux-x86_64/libgomp.so.1'
        pickFirst 'org/bytedeco/javacpp/linux-x86/libgomp.so.1'
    }
}
```
